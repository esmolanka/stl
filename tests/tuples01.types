
#eval (Int, String, Float)

#eval (Int, String)

#eval (Int,)

#eval Int × String × Float

#eval Int × String

#eval Int * String -> Float
