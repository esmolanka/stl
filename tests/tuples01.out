tests/tuples01.types:2:1-2:26:
  Normalised:
    ∃ ρ. (Int × String × Float × ρ)
  Kind:
    Type
tests/tuples01.types:4:1-4:19:
  Normalised:
    ∃ ρ. (Int × String × ρ)
  Kind:
    Type
tests/tuples01.types:6:1-6:11:
  Normalised:
    ∃ ρ. (Int × ρ)
  Kind:
    Type
tests/tuples01.types:8:1-8:27:
  Normalised:
    ∃ ρ. (Int × String × Float × ρ)
  Kind:
    Type
tests/tuples01.types:10:1-10:19:
  Normalised:
    ∃ ρ. (Int × String × ρ)
  Kind:
    Type
tests/tuples01.types:12:1-12:28:
  Normalised:
    (∃ ρ. (Int × String × ρ)) → Float
  Kind:
    Type